test/unit/domain/plan.test.js

```diff
@@ -0,0 +1,15 @@
+import Plan from '../../../lib/domain/plan';
+
+describe('Plan', () => {
+  it('sets the created date to the current date/time', async () => {
+    jest
+      .spyOn(global.Date, 'now')
+      .mockImplementationOnce(() =>
+        new Date('2020-05-14T12:01:58.000Z').valueOf()
+      );
```

Comment:
If we end up doing this a lot there is a nice library that makes it less typing called [MockDate](https://www.npmjs.com/package/mockdate).